_posts/2015-03-02-cos-post.md

```diff
@@ -0,0 +1,44 @@
+---
+title: New Resources at CareerOneStop
+layout: post
+image: 
```

Comment:
@kategarklavs you can either remove this line or change it to `image: ""` until we get an image to put here.